app/views/dashboard/_header.html.erb

```diff
@@ -1,14 +1,15 @@
   <div class="row header">
 
     <% if current_user %>
-      <h1><%= t('dashboard._header.dashboard') %></h1>
+      <h4><%= t('dashboard._header.dashboard') %></h4>&nbsp&nbsp
```

Comment:
If I'm not wrong, `&nbsp;` ends with a semicolon.

Reply:
yeah it's `&nbsp;`
cc: @gautamig54